lib/srv/ctx.go

```diff
@@ -500,6 +490,15 @@ func (c *ServerContext) GetEnv(key string) (string, bool) {
 	return val, ok
 }
 
+// SyncParentEnv is used to re-synchronize env vars after
+// parent context has been updated.
+func (c *ServerContext) SyncParentEnv() {
```

Comment:
Not really synchronization since you're only pushing data one way, more like `UpdateParentEnv()`, what do you think?